### PR TITLE
Bundle Google Cloud Storage jar

### PR DESCRIPTION
**Pull request by frankgh**
- Run performance tests for google cloud storage

[#162315299]

### CODE REVIEW

File: concourse/scripts/pxf-perf-multi-node.bash [hunk 1]
```diff
@@ -272,6 +281,59 @@ function assert_count_in_table {
...
+        <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem</value>
+    </property>
+    <property>
+        <name>fs.AbstractFileSystem.gs.impl</name>
+        <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS</value>
+    </property>
+</configuration>
+EOF
+     # Make a backup of core-site and update it with the S3 core-site
+    gpscp -u centos -f /tmp/segment_hosts /tmp/core-site.xml =:/tmp/core-site-patch.xml
```
Comment (lavjain) [inline concourse/scripts/pxf-perf-multi-node.bash]: Reminder here to use the new `pxf cluster sync` commands once it is merged to master.

File: concourse/scripts/pxf-perf-multi-node.bash [hunk 2]
```diff
@@ -341,6 +350,59 @@ EOF
...
+      <name>google.cloud.auth.service.account.enable</name>
+      <value>true</value>
+    </property>
+    <property>
+      <name>google.cloud.auth.service.account.json.keyfile</name>
+      <value>/tmp/gsc-ci-service-account.key.json</value>
+    </property>
+    <property>
+        <name>fs.gs.impl</name>
+        <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem</value>
```
Comment (shivzone) [inline concourse/scripts/pxf-perf-multi-node.bash]: Why should fs impl and filesystem class be explicitly set when defaultFS is already set ? Doesn't core-default internally map them accordingly ?

File: concourse/scripts/pxf-perf-multi-node.bash (hunk 1 above)
Comment (frankgh) [reply concourse/scripts/pxf-perf-multi-node.bash]: sync is being used now

File: concourse/scripts/pxf-perf-multi-node.bash (hunk 2 above)
Comment (frankgh) [reply concourse/scripts/pxf-perf-multi-node.bash]: Yes it does. It has been removed. Thanks for catching it!

**Pull request merged**